README.md

```diff
@@ -137,7 +95,8 @@ Also make sure to check out our [PowerShell-RFC repository](https://github.com/p
 [known issues]: https://github.com/PowerShell/PowerShell/tree/master/docs/KNOWNISSUES.md
 [GitHub issues]: https://github.com/PowerShell/PowerShell/issues
 [new issue]:https://github.com/PowerShell/PowerShell/issues/new
-[development]: https://github.com/PowerShell/PowerShell-Docker/tree/master/docs/development.md
+[development]: https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/development.md
```

Comment:
I don't see where you use this link

Reply:
Its used in the development and contributing section - this was just a dead link fix